OpenRA.Mods.Common/Activities/HarvesterDockSequence.cs

```diff
@@ -47,34 +47,54 @@ public HarvesterDockSequence(Actor self, Actor refinery, int dockAngle, bool isD
 
 		public override Activity Tick(Actor self)
 		{
+			if (ChildActivity != null)
+			{
+				ChildActivity = ActivityUtils.RunActivity(self, ChildActivity);
+				return this;
```

Comment:
Is the missing `if (ChildActivity != null)` check here intentional?

Reply:
No. Fixed.